ui/app/services/custom-messages.js

```diff
@@ -3,15 +3,17 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+import { action } from '@ember/object';
 import Service, { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
-
+import { TrackedObject } from 'tracked-built-ins';
 export default class CustomMessagesService extends Service {
   @service store;
   @service namespace;
   @service auth;
   @tracked messages = [];
   @tracked showMessageModal = true;
+  bannerDismissObject = new TrackedObject();
```

Comment:
This manages the state for each banner? The logic here is a little hard to follow because this name implies that we'd set to `true` if a banner was dismissed, but below it's set to false. 

Could we rename it to `bannerState` or switch the logic if you want this to be `bannerDismissState` so that `true` means a banner has been dismissed and therefore hidden? If this service is converted to typescript then we'd know it was an object, but also seeing `new TrackedObject();` gives us this information so I don't think that needs to be part of the variable name.

Changing the name also makes the template logic easier to follow because it reads as
```hbs
{{#if (get this.customMessages.bannerState bannerMessage.id)}}
// banner state is true so render
{{/if}}
```
OR 
```hbs
{{#unless (get this.customMessages.bannerDismissState bannerMessage.id)}}
// banner dismissed is false so render
{{/unless}